project/api/migrations/0028_auto_20190304_1834.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.7 on 2019-03-04 18:34
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('api', '0027_auto_20190226_1001'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='bill',
+            name='number',
+            field=models.CharField(max_length=20),
+        ),
+        migrations.AlterField(
+            model_name='bill',
+            name='source',
+            field=models.CharField(choices=[(b'sunlight', b'sunlight'), (b'propublica', b'propublica')], default=b'propublica', max_length=50),
```

Comment:
We are deprecating the sunlight API. Unless there is data in the database from Sunlight, we no longer need it as an option.

Reply:
Choices should be there in case we'd like to run admin panel functionality.
I'd keep it because debugging of potential problems for further developers might be really hard. I can comment appropriate line in the code about sunlight